addons/native_dialog/gtk_menu.c

```diff
@@ -387,9 +387,14 @@ static gboolean do_show_popup_menu(gpointer data)
 
    bool position_called = false;
    if (menu)
+      /* gtk_menu_popup_at_widget only exists in gtk newer than 3.22 */
+#if GTK_CHECK_VERSION(3, 22, 0)
       gtk_menu_popup_at_widget(args->menu->extra1, menu,  GDK_GRAVITY_SOUTH_WEST,
                               GDK_GRAVITY_NORTH_WEST,
                               NULL);
+#else
+      gtk_menu_popup(args->menu->extra1, menu, NULL, NULL, NULL, 1, 0);
```

Comment:
I tried this, and this actually completely locked up ex_menu. I got it to sort of work if the second argument was set to NULL, but I note that even on newer GTKs this code appears to be completely broken.

Let's change the second argument to NULL and call it a day, we'll fix the underlying issue at some later point.

Reply:
Weird, it worked on my machine with no hiccup. Anyway, I'm no expert on GTK programming :sweat_smile: